### PR TITLE
Add tracking for interested buyers

### PR DESCRIPTION
**Pull request by fatlotus**
Fairly simple- so that we can show how much interest there is in each listing.

Test URL: https://add-buyer-tracking-dot-hosted-caravel.appspot.com/

### CODE REVIEW

**Comment by fatlotus** [comment]
#95

File: caravel/templates/listings/fullpage.html [hunk 1]
```diff
@@ -47,6 +47,19 @@ <h3 class="panel-title">Manage Listing</h3>
...
+        {% if session["email"] in listing.buyers %}
+          <p class="alert alert-success">
+            You have successfully expressed interest in this listing.</p>
+        {% else %}
+          {% if listing.buyers %}
+          <p class="alert alert-info">
+            {% if listing.buyers|length != 1 %}
+              This listing has {{ listing.buyers|length }} inquries.
+            {% else %}
+              This listing has 1 inqury.
```
Comment (georgeteo) [inline caravel/templates/listings/fullpage.html]: Oh so if there is not listing.buyer, then it won't display anything? 

I guess my point was, do we want the default to an icon for 0 buyers?

Comment (fatlotus) [inline caravel/templates/listings/fullpage.html]: Ah. Yeah, it felt a little superfluous to show the alert when nobody's expressed interest.

Comment (georgeteo) [inline caravel/templates/listings/fullpage.html]: Ok cool. Just checking.

**Comment by fatlotus** [comment]
Looks okay?

**Comment by georgeteo** [comment]
Yup. Fix conflicts and merge away.

**Pull request merged**